services/app-api/src/handlers/configuration.ts

```diff
@@ -72,7 +72,7 @@ async function configurePostgres(
         return new Error('Failed to create Prisma Client')
     }
 
-    const client: PrismaClient = prismaResult
+    const client: PrismaClient = prismaResult as unknown as PrismaClient
```

Comment:
I was wondering why we're now casting to `unknown` -- I guess it's from the recent change to make `NewPrismaClient` return an `ExtendedPrismaClient`? I think we should probably try to stay away from giving up some of the type safety here, even though we know it's still a PrismaClient.

What if we change `ExtendedPrismaClient` to an intersection: 
```typescript 
type ExtendedPrismaClient = PrismaClient & ReturnType<typeof extendedPrismaClient>
```

I'd have to play with it but I think something along those lines would give `tsc` enough info to not need `unknown` here?

Reply:
@mojotalantikite It didn't like the intersection because the narrower PrismaClient also used everywhere else, the intersection type clashes with that. I just changed all the consumers and tests to the new type to workaround. Seem alright?